src/main.rs

```diff
@@ -95,7 +99,17 @@ Then try to restart Plume.
         exit(0);
     }).expect("Error setting Ctrl-c handler");
 
-    rocket::ignite()
+    let mut config = Config::active().unwrap();
+    config.set_address(env::var("ROCKET_ADDRESS").unwrap_or_else(|_| "localhost".to_owned())).unwrap();
+    config.set_port(env::var("ROCKET_PORT").ok().map(|s| s.parse::<u16>().unwrap()).unwrap_or(8000));
```

Comment:
Not really related, but we are saying in the docs it should be `7878` by default (even if it is not currently the case :grimacing:), maybe it is a good occasion to change it?

Reply:
it is indeed. Also do you know of other rocket env vars I might have forgotten?